mongodb-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/MongoDBBatchSink.java

```diff
@@ -57,7 +59,13 @@ public MongoDBBatchSink(MongoDBSinkConfig config) {
 
   @Override
   public void prepareRun(BatchSinkContext context) throws Exception {
-    context.addOutput(Output.of(config.referenceName, new MongoDBOutputFormatProvider(config)));
+    Configuration conf = new Configuration();
+    String path = conf.get(
+      "mapreduce.task.tmp.dir",
```

Comment:
This is copied from https://github.com/mongodb/mongo-hadoop/blob/master/core/src/main/java/com/mongodb/hadoop/output/MongoOutputCommitter.java#L236, it is using this path as the base path for the tmp dir for the mongo sink.